RELEASE-NOTES.md

```diff
@@ -1,11 +1,15 @@
 # Wikibase DataModel Serialization release notes
 
+## 2.4.0 (dev)
+
+* Dropped support for PHP 5.3 and PHP 5.4
+
 ## 2.3.0 (2017-02-07)
 
-* Improved performance of `StatementDeserializer`.
-* Improved type safety throughout the code.
-* Reworked `DeserializerBaseTest` into `DispatchableDeserializerTest`.
-* Reworked `SerializerBaseTest` into `DispatchableSerializerTest`.
+* Improved performance of `StatementDeserializer`
+* Improved type safety throughout the code
+* Renamed `DeserializerBaseTest` to `DispatchableDeserializerTest`
+* Renamed `SerializerBaseTest` to `DispatchableSerializerTest`
```

Comment:
Right... "reworked" is so ambiguous tho

Perhaps this should not even be here - if it is an implementation detail the users should not worry about (no breaking change) then why have it here?